server/server.js

```diff
@@ -0,0 +1,10 @@
+var express = require('express');
+var bodyParser = require('body-parser');
+
+var app = express();
+app.use(bodyParser.json());
+app.use(express.static(__dirname + '/../client'));
```

Comment:
express.static should be in middleware.js. Also, I'm not sure if that path is valid